src/NuGet.Clients/NuGet.VisualStudio.Interop/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("NuGet.VisualStudio.Interop")]
+[assembly: AssemblyDescription("NuGet/Visual Studio interop assembly")]
+
+// The version of this assembly should never be changed.
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
This assembly is somewhat special. It is loaded by the template wizard using its strong name. This file stayed unchanged for years. I'm restoring it from the before-migration-revision. It shouldn't have been touched at all.